mono/utils/win64.asm

```diff
@@ -38,6 +38,36 @@ __mono_current_ip:
 
 mono_context_get_current endP
 
+copy_stack_data_internal_win32_wrapper PROC PUBLIC
+;rcx MonoThreadInfo *
+;rdx MonoStackData *
+;r8 MonoBuiltinUnwindInfo *
+;r9 CopyStackDataFunc
+
```

Comment:
Probably should comment, that this is saving nonvolatile registers.

Reply:
Kinda seems like a job for RtlCaptureContext, but your code does seem smaller and sufficient.